php/class-utils.php

```diff
@@ -710,8 +713,8 @@ public static function purge_fragments() {
 	 */
 	public static function purge_fragment( $index ) {
```

Comment:
Can we purge_fragment after utilizing it (I guess at Media::down_sync_asset)? That way we will not allow peak behavior, and will not be dependent on the 'shutdown' action to happen

Reply:
@yuval-cloudinary 

We can. But as a principle, do we want to change things outside the scope of PR intent? Changing it will require additional items for QA and potentially delay the next release.

Reply:
I'm not sure about the intent. I looked at the title, and think that the current changes would not improve filestream management, as they would be closed anyway when the PHP execution is completed.
  It will improve fragment deletion, but under some edge cases, there would still be fragments left out.
  
  If this is out-of-scope, maybe we can leave a comment in the code to make sure the issue is known